src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2004,7 +2005,34 @@ bool entitiesQuery
   auto_ptr<DBClientCursor>  cursor;
   Query                     query(finalQuery.obj());
 
-  query.sort(BSON(ENT_CREATION_DATE << 1));
+  if (sorted == "")
```

Comment:
Not sure I understand this ...
sorted is a string, and if its empty then *query.sort()* is called and if it contains **anything at all**. then it is a vector of sortTokens?

First I thought it was some kind of boolean variable, telling us that it was either sorted or not.
Perhaps sortOrderList would be a better name?